tests/admin-ui-tests/utils.ts

```diff
@@ -21,6 +22,26 @@ const promiseSignal = (): Promise<void> & { resolve: () => void } => {
 };
 const projectRoot = findRootSync(process.cwd());
 
+export const makeGqlRequest = async (query: string, variables?: Record<string, any>) => {
+  try {
+    const { errors } = await fetch('http://localhost:3000/api/graphql', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        query,
+        variables,
+      }),
+    }).then(res => res.json());
+    if (errors) {
+      throw errors;
+    }
+  } catch (e) {
+    throw e;
+  }
```

Comment:
this is unnecessary?

Reply:
I maybe misunderstanding how error handling in async/await works, but wouldn't we miss out on network errors, or other non-gql related errors if this was omitted? 

I assumed this was the equivalent to: 
```typescript
const { errors } = await fetch('...').catch(e => {
  throw e
})

if (errors) { 
 throw errors 
} 
```